clueless/models.py

```diff
@@ -289,7 +301,10 @@ class Move(Action):
 
     def validate(self):
         #TODO: implement
-        return True
+        if (self.fromSpace.posX + 2 == self.toSpace.posX) or (self.fromSpace.posX - 2  == self.toSpace.posX) or (self.fromSpace.posY+ 2  == self.toSpace.posY) or (self.fromSpace.posY - 2  == self.toSpace.posY):
```

Comment:
I believe the right way to do this would be looking at the spaceWest, spaceEast, spaceNorth and spaceSouth attributes.  For example,
if(self.toSpace == self.fromSpace.spaceWest or self.toSpace == self.fromSpace.spaceNorth or etc....)